src/isIsogram.test.js

```diff
@@ -6,4 +6,34 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`should be returned true for ''`, () => {
```

Comment:
It is better to describe, not that it should return true for some exact word, but why it should return true.
Examples:
'should return true if entered string has no repeatings/empty string/repeatings with capitals etc'
Please review the rest of the tests.